develop/addons/schema-driven-forms/customising-form-behaviour/actions.rst

```diff
@@ -87,7 +87,7 @@ Updating button properties
 As with regular widgets, it is sometimes useful to set properties on
 buttons after they have been instantiated by *z3c.form*. One common
 requirement is to add a CSS class to the button. The standard edit form
-in*plone.directives.form* does this, for example, to add Plone’s
+in *plone.directives.form* does this, for example, to add Plone’s
```

Comment:
@svx I made the replacement here https://github.com/plone/documentation/pull/633. When date for the next release. I am still going through the whole section as stated here https://community.plone.org/t/clean-up-documentation-section-schema-driven-forms/2163 (there is much more stuff, that needs to be replaced).